docs/databases/connections/starburst.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Starburst
+---
+
+# Starburst
```

Comment:
@luizarakaki Anything we should add here?

Reply:
I think it is worth mentioning that this driver is Trino compatible. Given that Starburst is hosted Trino, I believe most people are looking for Trino and not Starburst.

We are keeping the Starburst name as they built the driver, but it should be clear that this is Trino compatible.